src/datasets/tasks/automatic_speech_recognition.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import ClassVar, Dict
+
+from ..features import Features, Value
+from .base import TaskTemplate
+
+
+@dataclass(frozen=True)
+class AutomaticSpeechRecognition(TaskTemplate):
+    task: str = "automatic-speech-recognition"
+    input_schema: ClassVar[Features] = Features({"audio_file": Value("string")})
```

Comment:
Maybe `audio_file_path` would be more explicit on what the column represent ?

Reply:
Also, paths are not portable between machines a priori. This is probably good enough for now, but at some point, we'll need to replace the `Value("string")` with an `Audio` or `Signal` feature!